### PR TITLE
ansible-scylla-monitoring: Pass cql credentials to scylla-monitoring

### PR DESCRIPTION
**Pull request by igorribeiroduarte**
Closes https://github.com/scylladb/scylla-ansible-roles/issues/227
Closes https://github.com/scylladb/scylla-ansible-roles/issues/93

### CODE REVIEW

File: ansible-scylla-monitoring/tasks/docker.yml [hunk 1]
```diff
@@ -68,6 +74,9 @@
         -R {{ scylla_monitoring_prom_rules_path }}/ \
         -b --storage.tsdb.retention.time={{ scylla_monitoring_retention }} \
         {{ scylla_monitoring_params | default('') }}
+  environment:
+    SCYLLA_USER: "{% if monitoring_cql_username is defined and monitoring_cql_password is defined %}{{ monitoring_cql_username }}{% else %}{% endif %}"
```
Comment (vladzcloudius) [inline ansible-scylla-monitoring/tasks/docker.yml:78]: Do you really need an `else` clause here and below?

File: ansible-scylla-monitoring/tasks/docker.yml [hunk 2]
```diff
@@ -123,4 +132,7 @@
         -b --storage.tsdb.retention.time={{ scylla_monitoring_retention }} \
         {{ scylla_monitoring_params | default('') }}
     become: true
+  environment:
+    SCYLLA_USER: "{% if monitoring_cql_username is defined and monitoring_cql_password is defined %}{{ monitoring_cql_username }}{% else %}{% endif %}"
```
Comment (vladzcloudius) [inline ansible-scylla-monitoring/tasks/docker.yml:136]: Same here.

File: ansible-scylla-monitoring/tasks/docker.yml (hunk 1 above)
Comment (igorribeiroduarte) [reply ansible-scylla-monitoring/tasks/docker.yml]: Nope. Fixed.

File: ansible-scylla-monitoring/tasks/docker.yml (hunk 2 above)
Comment (igorribeiroduarte) [reply ansible-scylla-monitoring/tasks/docker.yml]: Fixed.

**Comment by igorribeiroduarte** [comment]
> Very nice patch and well described patch. Pleasure to review!
Thanks!

> And one question: does this patch cover the non-docker Monitoring installation?
It doesn't.

> If not, please, clarify that in the patch description
Done

> let's have it as a follow up.
Ok, will start to work on it.

**Review by vladzcloudius — APPROVED** [review]

**Pull request merged**